css/qmwp-bargraph-scatterplot-timeline.css

```diff
@@ -382,3 +382,13 @@
     top: 50%;
     right: 40%;
 }
+
+.highcharts-tooltip > span {
+    z-index: 99 !important;
+    background-color: #FFF;
+    margin-left: -7px !important;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1308809522/issues/source?bid=2836188&fileBranchId=2951770#l389)